packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -72,7 +74,11 @@ export const ArticlePostCard = forwardRef(function PostCard(
       flagProps={{ pinnedAt, trending }}
       bookmarked={post.bookmarked && !showFeedback}
     >
-      <CardOverlay post={post} onPostCardClick={onPostCardClick} />
+      <CardOverlay
```

Comment:
We actually have `combinedClicks` which can do the same, as it's both the same function right?

Reply:
```ts
export const combinedClicks = (
  func: React.MouseEventHandler,
): CombinedClicks => {
  return {
    onAuxClick: (event) => event.button === 1 && func(event),
    onClick: func,
  };
};
```

I tried to use that, but it open modal if you aux clicked. You just want it to open new tab

Reply:
Hmm this just feels like we could easily miss one of the two which I'd like to avoid if that makes sense? (Eg if we need to add tracking to clicks or so)

Reply:
Discussed this in Gather. It re-uses the same top function, and because of prop-drilling and re-creating of the functions, it's the easiest way to do it now.